pyproject.toml

```diff
@@ -0,0 +1,59 @@
+[project]
+name = "head_direction"
+version = "0.1.0"
+authors = [
+    { name = "Mikkel Lepperod", email = "mikkel@simula.no" },
+    { name = "Alessio Buccino", email = "alessiop.buccino@gmail.com" },
+]
+
+description = "Compute spatial maps for neural data."
+readme = "README.md"
+requires-python = ">=3.10"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+]
+
+dependencies = [
+    "numpy<2",
```

Comment:
Won't this quickly create a conflict since many packages now are based on numpy>=2?